src/test/java/com/foundation/trello/step/ManagerStepDef.java

```diff
@@ -83,6 +84,7 @@ public void getStatusCodeAsResponse(int statusCode) {
      */
     @And("I verify the response schema whit {string}")
     public void iGetASchema(String schema) {
```

Comment:
Put a name more descriptive of parameter schema please

Reply:
I fixed it.